src/app/services/cli.ts

```
@@ -0,0 +1,107 @@
+    // Get package.json data
+    const packageJsonPath = path.resolve(basePath, "package.json");
+    const packageJson = JSON.parse(readFileSync(packageJsonPath, "utf-8"));;
+
+    for (let i = 0; i < args.length; i++) {
+        const arg = args[i];
+
+        switch (arg) {
+            case "--help":
+            case "-h":
+                printHelp();
+                process.exit(0);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)